packages/tooling/__tests__/lib/find-schema-definition.test.js

```diff
@@ -1,9 +1,39 @@
+const petstore = require('@readme/oas-examples/3.0/json/petstore.json');
 const findSchemaDefinition = require('../../src/lib/find-schema-definition');
 
+test('should return a definition for a given ref', () => {
+  expect(findSchemaDefinition('#/components/schemas/Pet', petstore)).toStrictEqual(petstore.components.schemas.Pet);
+});
+
+test('should return a definition for a given ref that is escaped', () => {
+  expect(
+    findSchemaDefinition('#/components/schemas/Pet~1Error', {
```

Comment:
Is this pattern of escaping actually what happens in openapi? Is this supported? `Pet~1Error`

Reply:
Yeah, to have a a schema, `Pet/Error`, with a slash in it you need to escape the `/` as `~1`, otherwise trying to access `#/components/schemas/Pet/Error` would try to load:

```
components: { schemas: { Pet: { Error: { ... } } } }
``` 

[jsonpointer](https://www.npmjs.com/package/jsonpointer) handles this all for us.

https://swagger.io/docs/specification/using-ref/#escape